index.html

```diff
@@ -1,382 +1,333 @@
-                <p>Let's Connect</p>
-                <div class="list-nav">
-                  <ul class="list-items">
-                    <li>
-                      <a href="">
-                        <img
-                          src="./images/twitter.png"
-                          alt="my social handle images"
-                        />
-                      </a>
-                    </li>
-                    <li>
-                      <a href="">
-                        <img
-                          src="./images/Linkedin-icon.svg"
-                          alt="my social handle images"
-                        />
-                      </a>
-                    </li>
-                    <li>
-                      <a href="">
-                        <img
-                          src="./images/medium.png"
-                          alt="my social handle images"
-                        />
-                      </a>
-                    </li>
-                    <li>
-                      <a href="">
-                        <img
-                          src="./images/github.png"
-                          alt="my social handle images"
-                        />
-                      </a>
-                    </li>
-                    <li>
-                      <a href=""
-                        ><img
-                          src="./images/angellist.png"
-                          alt="my social handle images"
-                        />
-                      </a>
-                    </li>
-                  </ul>
-                </div>
-              </div>
+        <div class="desktop-tools">
+          <ul class="flex">
+            <li>
+              <a href="#">Portfolio</a>
+            </li>
+            <li>
+              <a href="#">About</a>
+            </li>
+            <li>
+              <a href="#">Contact</a>
+            </li>
+          </ul>
+        </div>
+
+        <nav class="nav-link">
+          <div class="nav-menu">
+            <div class="ambuger-menu">
+              <span class="bar"></span>
+              <span class="bar"></span>
+              <span class="bar"></span>
             </div>
           </div>
-        </section>
-      </main>
-    </div>
-    <section class="project-section">
-      <div class="project-container">
-        <div class="project-card flex column">
-          <img
-            src="./images/Snapshoot Portfolio.svg"
-            alt="project card image"
-          />
-          <div class="prject-card-info flex column">
-            <h2 class="project-title">Tonic</h2>
-            <div class="project-meta flex">
-              <p class="bold">Canopy</p>
-              <div class="round"></div>
-              <p>Backend</p>
-              <div class="round"></div>
-              <p>2015</p>
-            </div>
-            <div class="project-note">
+        </nav>
+      </div>
+    </header>
+    <main>
```

Comment:
Lovely job @EnixWord 👌 but kindly fix this section as previous reviewers requested. The background image is distorted because this container's height does not match the Figma template. Please notice the difference of the sizes below.
Your design:
<img width="1440" alt="Screenshot 2022-12-17 at 10 57 12 PM" src="https://user-images.githubusercontent.com/25887253/208264430-5283de2a-cbdd-44e0-9c83-3f27d2f86bf1.png">
Figma template:
<img width="1440" alt="Screenshot 2022-12-17 at 10 57 26 PM" src="https://user-images.githubusercontent.com/25887253/208264433-9847567e-d9e6-4c67-9451-71fae6de9baa.png">